tf_big/cc/kernels/big_tensor.h

```diff
@@ -42,12 +42,14 @@ struct NumTraits<mpz_class> : GenericNumTraits<mpz_class> {
 
 typedef Matrix<mpz_class, Dynamic, Dynamic> MatrixXm;
 
+namespace tfbig {
```

Comment:
wondering if it makes sense to pull `BigTensor` code out of the kernels directory..?

Reply:
Yeah probably! https://github.com/tf-encrypted/tf-big/issues/15